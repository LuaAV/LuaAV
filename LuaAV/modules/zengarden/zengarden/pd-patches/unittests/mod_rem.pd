#N canvas 302 22 760 533 10;
#X obj 149 35 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 184 27 loadbang;
#X obj 164 77 t b b b b;
#X obj 392 145 f 10;
#X obj 392 223 % 3;
#X obj 446 224 mod 3;
#X obj 246 309 print mod;
#X obj 183 308 print %;
#X obj 313 222 mod -3;
#X obj 259 221 % -3;
#X obj 259 143 f 10;
#X obj 139 142 f -10;
#X obj 193 221 mod 3;
#X obj 139 220 % 3;
#X obj 35 142 f -10;
#X obj 35 220 % -3;
#X obj 80 221 mod -3;
#X obj 399 184 t f f;
#X obj 272 179 t f f;
#X obj 46 178 t f f;
#X obj 149 180 t f f;
#X text 293 35 This test ensures that modulus and remainder objects
are working properly.;
#X connect 0 0 2 0;
#X connect 1 0 2 0;
#X connect 2 0 14 0;
#X connect 2 1 11 0;
#X connect 2 2 10 0;
#X connect 2 3 3 0;
#X connect 3 0 17 0;
#X connect 4 0 7 0;
#X connect 5 0 6 0;
#X connect 8 0 6 0;
#X connect 9 0 7 0;
#X connect 10 0 18 0;
#X connect 11 0 20 0;
#X connect 12 0 6 0;
#X connect 13 0 7 0;
#X connect 14 0 19 0;
#X connect 15 0 7 0;
#X connect 16 0 6 0;
#X connect 17 0 4 0;
#X connect 17 1 5 0;
#X connect 18 0 9 0;
#X connect 18 1 8 0;
#X connect 19 0 15 0;
#X connect 19 1 16 0;
#X connect 20 0 13 0;
#X connect 20 1 12 0;
