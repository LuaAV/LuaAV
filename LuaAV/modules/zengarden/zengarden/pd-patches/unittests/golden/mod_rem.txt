mod: 1
%: 1
mod: 1
%: 1
mod: 2
%: -1
mod: 2
%: -1